src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import usersFromServer from '../../api/users';
+
+interface TodoFormProps {
+  title: string;
+  userId: number;
+  errorTitle: string;
+  errorUserId: string;
+  onTitleChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onUserChange: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  onSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
+}
+
+export const TodoForm: React.FC<TodoFormProps> = ({
+  onTitleChange,
+  onSubmit,
+  errorUserId,
+  onUserChange,
+  title,
+  errorTitle,
+  userId,
+}) => {
+  return (
+    <form action="/api/todos" method="POST" onSubmit={onSubmit}>
+      <div className="field">
+        <label className="label" htmlFor={'title'}>
+          Title:
+        </label>
+        <input
+          placeholder="Enter a title"
+          type="text"
+          id="title"
+          data-cy="titleInput"
+          value={title}
+          onChange={onTitleChange}
+        />
+        {errorTitle && <span className="error">{errorTitle}</span>}
+      </div>
+
+      <div className="field">
+        <label className="label" htmlFor={'title'}>
+          User:
+        </label>
+        <select
+          data-cy="userSelect"
+          required
```

Comment:
If you added label with htmlFor path  you also you also need to add an id property to select with the same value as you set in label htmlFor
```suggestion
          required
          id='user'
```

Reply:
> If you added label with htmlFor path you also you also need to add an id property to select with the same value as you set in label htmlFor

Good point